cl/phase1/network/services/aggregate_and_proof_service.go

```diff
@@ -215,8 +215,12 @@ func (a *aggregateAndProofServiceImpl) ProcessMessage(
 	// for this specific request, collect data for potential peer banning or gossip publishing
 	aggregateVerificationData.GossipData = aggregateAndProof.GossipData
 
+	if aggregateAndProof.ImmediateProcess {
+		return a.batchSignatureVerifier.ImmediateVerification(aggregateVerificationData)
```

Comment:
Same here. maybe return `ErrIgnore` ? 🤔

Reply:
Same as above, I think it makes the code much easier anyway to read anyway